pkg/models/user_info.go

```diff
@@ -1,8 +1,16 @@
 package models
 
 // UserInfo is the model for personal details of a user
-type UserInfo struct {
-	CommonName string
-	Email      EmailAddress
-	EuaUserID  string
+type UserInfo struct { //TODO, this needs to be updated to allow for new fields from OKTA
```

Comment:
I think this `TODO` can probably go now that we're pulling `FirstName`, `LastName`, etc?